src/components/Navbar.jsx

```diff
@@ -15,16 +17,21 @@ const Navbar = () => {
     },
   ];
   return (
-    <nav>
-      <ul>
-        {links.map((link) => (
-          <li key={link.id}>
-            <NavLink className="navlink" exact="true" to={link.path}>
-              {link.text}
-            </NavLink>
-          </li>
-        ))}
-      </ul>
+    <nav className="navbar">
+      <div className="logo">
+        <ul className="items">
+          {links.map((link) => (
+            <li key={link.id}>
+              <NavLink className="navlink" exact="true" to={link.path}>
+                {link.text}
+              </NavLink>
+            </li>
+          ))}
+        </ul>
+      </div>
+      <div className="avatar">
+        <FaUserAlt />
+      </div>
     </nav>
```

Comment:
- [ ] Your design is missing `Bookstore CMS` title from the design. Please make sure to add that and it should match with the zeplin design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/185755052-ea54d6e0-93a7-488e-83b4-2cef58cfbbb7.png)

### Zeplin design

![image](https://user-images.githubusercontent.com/83803414/185755067-ab01e131-9469-46af-8730-5b0029267c01.png)